pages/index.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div class="mainDiv">
+    <div class="titletext">Add Tasks</div>
+    <div class="spaceBetween marginTop28 marginBottom34">
+      <div>
+        <button @click="createTask()">{{ $t('create') }}</button>
+      </div>
+      <div>
+        <button class="marginLeft19">All</button>
+        <button class="marginLeft19">Incomplete</button>
+        <button class="marginLeft19">Complete</button>
+      </div>
+    </div>
+    <div class="listDiv">
+      <div v-if="showAddCard" class="card">
+        <textarea v-model="task.description"></textarea>
+        <div class="flexBox">
+          <button class="addButton" @click="addTask">Add Task</button>
+          <div class="marginTop13" @click="clearField()">
+            <DeleteButton />
+          </div>
+        </div>
+      </div>
+      <div v-for="(task, index) in taskData" :key="index">
+        <TaskCard :card-data="task" />
+      </div>
+    </div>
+    <div class="wrapper">
+      <div class="content">
+        <div class="centerItem">
+          <img :src="noTaskLogo" />
+        </div>
+        <div class="infoText marginTop32">
+          You didn’t add any task. Please, add one.
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+import DeleteButton from '../components/buttons/DeleteButton.vue';
+import noTaskLogo from '../assets/svg/noTask.svg';
+export default {
+  name: 'IndexPage',
+  components: { DeleteButton },
+  layout: 'default',
+  data: () => ({
+    task: {
+      description: '',
+      createdAt: null,
+    },
+    taskData: [],
+    showAddCard: false,
+    noTaskLogo,
+  }),
+  computed: {
+    ...mapGetters(['getTodoList', 'getFilteredList']),
+  },
+  created() {
+    this.$nuxt.$on('searchOn', () => {
+      this.taskData = this.getFilteredList;
+    });
+    console.log(this.getTodoList);
```

Comment:
We can remove the unnecessary console log

Reply:
Resolved